components/NavBar.js

```diff
@@ -0,0 +1,114 @@
+import styled from 'styled-components';
+ 
+const NavBarContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 46px 87px;
+`;
+
+const NavGroupContainer = styled.div`
+  display: flex;
+  gap: 28px;
+  align-items: center;
+`;
+
+const StyledImage = styled.img`
+  margin-right: 18px;
+`;
+
+const SimpleLinkText = styled.span`
+  font-weight: bold;
+  color: ${span => span.theme.colors.secondary};
+
+  &:hover {
+    background: ${span => span.theme.colors.primaryGradient};
+    -webkit-background-clip: text;
```

Comment:
```suggestion
  color: ${p=> p.theme.colors.secondary};

  &:hover {
    background: ${p=> p.theme.colors.primaryGradient};
    -webkit-background-clip: text;
```
The `p` stands for "props" not "paragraph" :P

Reply:
fixed!